### PR TITLE
playbackRate selection and expandable player

### LINKED ISSUE #253: Use Android App Bundle
Change gradle from assemble to bundle:

`    gradle(
      task: "assemble",
      build_type: "Release"
    )`

will reduce app size by a lot

### PR DESCRIPTION
**Pull request by ovbm**
fixes #253

### CODE REVIEW

File: src/components/AudioPlayer/ExpandedControls.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+      {Platform.OS === 'android' && (
+        <View
+          style={{
+            position: 'absolute',
+            top: -12,
+            zIndex: 1,
+            width: '100%',
+          }}
+          onLayout={e => setExpandedControlsWidth(e.nativeEvent.layout.width)}>
+          <Svg height={12} width={expandedControlsWidth}>
```
Comment (ovbm) [inline src/components/AudioPlayer/ExpandedControls.js:39]: maybe expandedControlsWidth is unnecessary, react-native svg supports` width=100%`

**Comment by trm217** [comment]
Dragging the progressbar causes the colored bar to jump, seems like a delayed update?

https://user-images.githubusercontent.com/30313631/159904078-b997aa36-0db1-4779-9cb8-cb375632f72b.mp4

**Comment by ovbm** [comment]
> Dragging the progressbar causes the colored bar to jump, seems like a delayed update?
> 
>  RPReplay_Final1648116606.mp4

Don't understand what I'm looking at. What are you expecting to happen?

**Comment by trm217** [comment]
Seems like a bug to me. Of course it works, but it looks funny.

File: src/components/AudioPlayer/ExpandedControls.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+                }}>
+                <Text
+                  style={[
+                    {
+                      fontWeight:
+                        rate.speed === playbackRate ? 'bold' : 'normal',
+                      color: colors.text,
+                    },
+                    styles.rateSelector,
+                  ]}>{`${rate.label}`}</Text>
```
Comment (ovbm) [inline src/components/AudioPlayer/ExpandedControls.js:152]:
```suggestion
                  ]}>{rate.label}</Text>
```

**Review by tpreusse — APPROVED** [review]

**Pull request merged**